plugins/orchestrator/src/components/WorkflowDialog/WorkflowDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import {
+  Box,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+import { WorkflowEditor } from '../WorkflowEditor';
+import { WorkflowEditorView } from '../WorkflowEditor/WorkflowEditor';
+
+type OrchestratorWorkflowDialogProps = {
+  workflowId: string;
+  title: string;
+  open: boolean;
+  close: () => void;
+} & WorkflowEditorView;
+
+const useStyles = makeStyles(_theme => ({
+  editor: {
+    height: '600px',
+    marginBottom: 20,
+  },
+  closeBtn: {
+    position: 'absolute',
+    right: 8,
+    top: 8,
+  },
+}));
+
+export const WorkflowDialog = (
+  props: OrchestratorWorkflowDialogProps,
+): JSX.Element | null => {
+  const { workflowId, title, open, close } = props;
+  const classes = useStyles();
+
+  return (
+    <Dialog fullWidth maxWidth="lg" onClose={_ => close()} open={open}>
```

Comment:
```suggestion
  onClose={close}
```

Reply:
There is a refactoring going on and this component will likely be replaced with new code.